src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -75,7 +75,15 @@ const formValidation = (values) => {
 
   const safeNonceValidation = minValue(0, true)(safeNonce)
 
-  const safeTxGasValidation = minValue(0, true)(safeTxGas)
+  const safeTxGasValidation = composeValidators(minValue(0, true), (value: string) => {
+    if (!value) {
+      return
+    }
+
+    if (Number(value) > Number(ethGasLimit)) {
+      return 'Bigger than Ethereum gas limit.'
```

Comment:
shouldn't we avoid using `ethereum`? I mean, we now have support xDai, volta, etc.